ci/jenkins/Jenkins_steps.groovy

```diff
@@ -756,6 +802,22 @@ def test_unix_python3_gpu() {
     }]
 }
 
+def test_unix_python3_gpu_no_tvm_op() {
+    return ['Python3: GPU TVM_OP OFF': {
+      node(NODE_LINUX_GPU) {
+        ws('workspace/ut-python3-gpu-no-tvm-op') {
+          try {
+            utils.unpack_and_init('gpu_no_tvm_op', mx_lib_cython_no_tvm_op)
```

Comment:
The build corresponding to this test packs library set `mx_lib_cpp_examples_no_tvm_op `
https://github.com/apache/incubator-mxnet/pull/16450/files#diff-bb61a49bf10098c4c42879f1632fdb40R273

We should use the same library set while unpacking. A new library set is not required.
Suggested action: Replace `mx_lib_cython_no_tvm_op ` with `mx_lib_cpp_examples_no_tvm_op `